openassessment/workflow/models.py

```diff
@@ -613,8 +613,37 @@ class TeamAssessmentWorkflow(AssessmentWorkflow):
     """
     Extends AssessmentWorkflow to support team based assessments.
     """
+    # Only staff assessments are supported for teams
+    STEPS = ['staff']
+
     team_submission_uuid = models.CharField(max_length=128, unique=True, null=False)
 
+    @classmethod
+    @transaction.atomic
+    def start_workflow(cls, team_submission_uuid):  # pylint: disable=arguments-differ
+        """ Start a team workflow """
+        # TODO - Implement in https://openedx.atlassian.net/browse/EDUCATOR-4986
+        workflow = {'team_submission_uuid': team_submission_uuid}
+        return workflow
+
+    @classmethod
+    def get_by_team_submission_uuid(cls, team_submission_uuid):
+        try:
+            return cls.objects.get(team_submission_uuid=team_submission_uuid)
+        except cls.DoesNotExist:
+            return None
+        except DatabaseError as exc:
+            message = (
+                u"Error finding workflow for team submission UUID {uuid} due to error: {exc}."
```

Comment:
Nit (which applies to many string literals below): you don't have to use that `u` prefix, because python 3 (although it's still valid syntax).